### ISSUE TITLE
looking for "biometric device" on dashboard, but did not find

### ISSUE THREAD

**Issue by sharukhkhan234** [horilla-opensource/horilla#192]
# Bug Report

## Description

_[A brief description of the bug or issue you're encountering.]_

## Steps to Reproduce

1. _[Detailed step-by-step instructions to reproduce the issue.]_

## Expected Behavior

_[What you expected to happen.]_

## Actual Behavior

_[What actually happened.]_

## Screenshots

_[If applicable, provide screenshots illustrating the issue.]_

## Environment

- **Django Version:** _[Django version you're using, e.g., 3.2.1]_
- **Python Version:** _[Python version, e.g., 3.9]_
- **Operating System:** _[Your OS, e.g., Windows 10, Ubuntu 20.04]_
- **Browser:** _[If applicable, specify the browser and version you're using.]_

## Additional Information

_[Any other relevant information, logs, error messages, or context that might help in understanding and fixing the issue.]_

## Possible Solution

_[If you have any ideas or suggestions on how to fix the issue, feel free to provide them here. This is optional.]_

## Labels

_[If your project uses labels, suggest any labels that might apply to this issue, such as 'bug', 'needs investigation', etc.]_

## Priority

_[Specify the priority level for this issue, such as 'high', 'medium', 'low', etc.]_

## Assignees

_[If you want to suggest an assignee or tag a specific person to look into this issue, mention their GitHub username here.]_

## Related Issues

_[If there are any related issues or pull requests, mention them here. This is optional.]_

**Note:** _Remember to search through existing issues before submitting a new one to ensure that the issue hasn't been reported already. Provide as much information as possible to help the maintainers understand and address the problem effectively._

**Comment by sharukhkhan234** [comment]
plz anyone help me for this issue

**Comment by anoninthekop** [comment]
There is a separate branch named Biometric. You will have to run that branch although it is many commits behind master.

**Comment by sharukhkhan234** [comment]
> There is a separate branch named Biometric. You will have to run that branch although it is many commits behind master.
from where I have to run that branch, plz share complete details

**Comment by sharukhkhan234** [comment]
I am waiting for your response,

**Comment by horilla-opensource** [comment]
Hi @sharukhkhan234 ,
Sorry the delay in the response.
I hope you got the biometric module. Please let us know if you face any issues.

With Regards,
**Team Horilla**

**Comment by sharukhkhan234** [comment]
Explanation!
I just followed the commands which are given by horilla. However it
installed and I signed in. As I saw biometric device option on the site,
but when I got install, I didn't saw biometric device option over there.
Therefore I need guidance that how I can display biometric device on Left
menu bar. I didn't make any changes in the code. It's requested to guide
me, that how I can see that app
